recipes/h5pyd/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "h5pyd" %}
+{% set version = "0.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/h5pyd-{{ version }}.tar.gz
+  sha256: 168a50f23c493193dcbdbc2e6f10dd2f9b9a8ab45f289e37685fc138821f82eb
+
+build:
+  entry_points:
+    - hsinfo = h5pyd._apps.hsinfo:main
+    - hsls = h5pyd._apps.hsls:main
+    - hstouch = h5pyd._apps.hstouch:main
+    - hsacl = h5pyd._apps.hsacl:main
+    - hsdel = h5pyd._apps.hsdel:main
+    - hsrm = h5pyd._apps.hsdel:main
+    - hsget = h5pyd._apps.hsget:main
+    - hsload = h5pyd._apps.hsload:main
+    - hsconfigure = h5pyd._apps.hsconfigure:main
+    - hscopy = h5pyd._apps.hscopy:main
+    - hscp = h5pyd._apps.hscopy:main
+    - hsmv = h5pyd._apps.hsmv:main
+    - hsdiff = h5pyd._apps.hsdiff:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pkgconfig
+    - pip
+  run:
+    - python >=3.6
+    - msrestazure
+    - google-auth <2.0dev
+    - adal
+    - cryptography
+    - google-api-python-client
+    - google-auth-oauthlib
+    - requests-unixsocket
+    - numpy >=1.17.3
+    - pytz
+    - pyjwt
+
+test:
+  imports:
+    - h5pyd
+  commands:
+    - pip check
+    - hsinfo --help
+    - hsls --help
+    - hstouch --help
+    - hsacl --help
+    - hsdel --help
+    - hsrm --help
+    - hsget --help
+    - hsload --help
+    - hscopy --help
+    - hscp --help
+    - hsmv --help
+    - hsdiff --help
+  requires:
+    - pip
+    - h5py
+
+about:
+  home: http://github.com/HDFGroup/h5pyd
+  summary: h5py compatible client lib for HDF REST API
+  license: BSD-4-Clause
```

Comment:
I think the license is a custom one: https://github.com/HDFGroup/h5pyd/blob/master/COPYING

Reply:
Looks like it. Even though they declare it as BSD in the metadata :-/